api/public_api.swagger.json

```diff
@@ -5677,8 +5677,8 @@
       "tags": ["Organizations", "Invitations", "Policies", "Features"]
     },
     {
-      "name": "PRIVATE KEYS",
-      "tags": ["Wallets", "Signers", "Private Keys", "Private Key Tags"]
+      "name": "WALLETS AND PRIVATE KEYS",
+      "tags": ["Wallets", "Signing", "Private Keys", "Private Key Tags"]
```

Comment:
will need to double check that linking/etc all works as expected on the docs page. it should! 🤞

Reply:
I had to search/replace all "Signers" and replace with "signing", we had an example using the signers namespace!